.ci/jsoref-spellchecker/whitelist.words

```diff
@@ -365,6 +365,7 @@ edb
 edef
 edf
 edu
+Eghj
```

Comment:
I am not seeing where this word came from in PR.

Reply:
It is a leack from your commit on new arguments

Reply:
I see it now, thanks.